conf/machine/iot2050.conf

```diff
@@ -18,3 +18,5 @@ IMAGE_TYPE ?= "wic-img"
 WKS_FILE ?= "iot2050.wks.in"
 
 IMAGE_INSTALL += "u-boot-script"
+
+IOT2050_VARIANTS ??= "PG1"
```

Comment:
This is better modeled via separate u-boot recipes and, thus, differences in PREFERRED_PROVIDER_u-boot-iot2050.